report_qweb_field_converter/models/ir_qweb.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Quartile Limited (https://www.quartile.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import json
+import logging
+
+from odoo import api, models
+
+_logger = logging.getLogger(__name__)
+
+
+class IrQweb(models.AbstractModel):
+    _inherit = "ir.qweb"
+
+    @api.model
+    def _get_field(
+        self, record, field_name, expression, tagName, field_options, values
+    ):
+        report_type = values.get("report_type")
+        if not report_type or report_type != "pdf":
+            return super()._get_field(
+                record, field_name, expression, tagName, field_options, values
+            )
+        qweb_recs = self.env["qweb.field.converter"].search(
```

Comment:
```suggestion
        qweb_recs = self.sudo().env["qweb.field.converter"].search(
```
We should either do this or make `qweb.field.converter` globally accessible. Please check and suggest which way is better. Portal users get 403 forbidden error when accessing their documents otherwise.